<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateAccountnext</class>
 <widget class="QDialog" name="UpdateAccountnext">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#UpdateAccountnext{
	background-image: url(:/bg_updatenext.jpeg);
}
#frame_2{
background-image: url(:/icon.png);
}
#label{
color:white;
}
#label_2{
color:white;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>461</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color:rgba(82, 82, 82,0.5);
color:white;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>YOUR UPDATE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genderEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="privacyEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Private/Public</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="text">
     <string>UPDATE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dobEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FVF Fernando 08</family>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Date of Birth</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2{
bacfground:transparent;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FVF Fernando 08</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Instagram</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
